packages/orchestration/src/facade.js

```diff
@@ -127,8 +130,13 @@ const makeRemoteChainFacade = (name, { orchestration, timer, zcf, zone }) => {
 
       const address = await E(icaAccount).getAddress();
 
-      // FIXME look up real values
-      const bondDenom = name;
+      const [{ denom: bondDenom }] = chainInfo.stakingTokens || [
+        {
+          denom: null,
+        },
+      ];
+      assert(bondDenom, 'missing bondDenom');
```

Comment:
Mentioned [here](https://github.com/Agoric/agoric-sdk/pull/9452/files#r1628013682) - but i don't expect we'll see an array of staking staking tokens for anyone using `cosmos.staking.v1beta1`